owl/SOMA-SAY.owl

```diff
@@ -80,13 +166,69 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#CausalProcessRole -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#CausalProcessRole">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Role"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#classifies"/>
+                <owl:allValuesFrom rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#ExecutableSchematicTheory"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">A role that denotes physical processes causing a motion such as the force interaction of a tennis racket hitting a ball and setting it into motion.</rdfs:comment>
```

Comment:
the comment is a bit confusing. For me, "A role that denotes physical processes" implies that the role is used to classify processes. Which would not be consistent as processes are no objects. And the axiomatization states that the role classifies some theory.
Maybe could be clarified?